plugin/evm/atomic/atomic_trie.go

```diff
@@ -108,20 +109,20 @@ type AtomicTrieIterator interface {
 
 	// AtomicOps returns a map of blockchainIDs to the set of atomic requests
 	// for that blockchainID at the current block number
-	AtomicOps() *atomic.Requests
+	AtomicOps() *avalancheatomic.Requests
 
 	// Error returns error, if any encountered during this iteration
 	Error() error
 }
 
 // atomicTrie implements the AtomicTrie interface
 type atomicTrie struct {
-	commitInterval      uint64            // commit interval, same as commitHeightInterval by default
-	metadataDB          database.Database // Underlying database containing the atomic trie metadata
-	trieDB              *triedb.Database  // Trie database
-	lastCommittedRoot   common.Hash       // trie root of the most recent commit
-	lastCommittedHeight uint64            // index height of the most recent commit
-	lastAcceptedRoot    common.Hash       // most recent trie root passed to accept trie or the root of the atomic trie on intialization.
+	commitInterval      uint64                     // commit interval, same as commitHeightInterval by default
+	metadataDB          avalanchedatabase.Database // Underlying database containing the atomic trie metadata
+	trieDB              *triedb.Database           // Trie database
+	lastCommittedRoot   common.Hash                // trie root of the most recent commit
+	lastCommittedHeight uint64                     // index height of the most recent commit
+	lastAcceptedRoot    common.Hash                // most recent trie root passed to accept trie or the root of the atomic trie on intialization.
```

Comment:
Note for the future/new code (to myself as well 😄) - we should write comments above fields and not to the the right, otherwise gofmt causes unneeded git diffs on other fields as soon as a field name or type gets longer/shorter than the longest/smallest.